test/check-application

```diff
@@ -31,21 +31,51 @@ class TestNavigator(testlib.MachineCase):
 
         self.login_and_go("/navigator")
         # expected heading
-        b.wait_text(".pf-c-card__title", "Directories & files")
+        b.wait_text("#navigator-card-header", "Directories & files")
+        b.wait_text("#sidebar-card-header", "Current directory")
 
         # new files are auto-detected
-        m.execute("touch /home/admin/newfile")
+        m.execute("touch -t 9407031305 /home/admin/newfile")
         b.wait_visible("[data-item='newfile']")
 
         # new directories are auto-detected
-        m.execute("mkdir /home/admin/newdir")
+        m.execute("mkdir /home/admin/newdir; touch -t 9407031500 /home/admin/newfile /home/admin/newdir")
         b.wait_visible("[data-item='newdir']")
 
         # hidden files are not displayed
         m.execute("touch /home/admin/.hiddenfile /home/admin/not-hidden")
         b.wait_visible("[data-item='not-hidden']")
         b.wait_not_present("[data-item='.hiddenfile']")
 
+        # file sidebar information
+        b.click("button:contains('newfile')")
+        b.wait_text("#sidebar-card-header", "newfile")
+        b.wait_text("#description-list-owner dd", "root")
+        b.wait_text("#description-list-group dd", "root")
+        b.wait_text("#description-list-size dd", "0 B")
+        # Packit test VMs have different timezone
+        b.wait(lambda: b.text("#description-list-last-modified dd") in ["Jul 3, 1994, 5:00 PM", "Jul 3, 1994, 3:00 PM"])
```

Comment:
Now the question is: Is that supposed to be in the server's or the browser's time zone? getFormattedDateTime() is browser time AFAIK. In packit, the browser runs in the test VM, in our CI it runs outside (with TZ=UTC, I believe). I think cockpit's timer tests do some browser time testing, they might have some inspiration.

Reply:
it should be the servers timezone according to my logic :)